db/schema.rb

```diff
@@ -69,6 +69,18 @@
     t.index ["name"], name: "index_backgrounds_on_name", unique: true
   end
 
+  create_table "backgrounds__dropped_cols_backup", id: false, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
```

Comment:
these `__dropped_cols_backup` shouldn't be added, right? Maybe you could reset this and just add the new fields manually

Reply:
Ah good catch. I just deleted all those in staging and I'll delete them locally. FYI, `_table_name` are ignored by `schema.rb` generation.